LibNFS.xs

```diff
@@ -983,6 +983,50 @@ static void _set_unix_authn(pTHX_ struct nfs_context* nfs, SV* value_sv) {
     nfs_set_auth(nfs, auth);
 }
 
+#ifdef NLNFS_NFS_SET_READDIR_MAX_BUFFER_SIZE
+static void _set_readdir_max_buffer_size(pTHX_ struct nfs_context* nfs, SV* value_sv) {
+
+    uint32_t dircount, maxcount;
+
+    if (SvROK(value_sv) && (SvTYPE(SvRV(value_sv)) == SVt_PVAV)) {
+        #define MAX_BUFFER_SIZE_ARRAY_SIZE 2
+        AV* av = (AV*) SvRV(value_sv);
+        uint32_t nums_count = 1 + av_len(av);
+        if (nums_count != MAX_BUFFER_SIZE_ARRAY_SIZE) croak("“%s” must contain exactly %d numbers", "readdir_buffer", MAX_BUFFER_SIZE_ARRAY_SIZE);
+
+        uint32_t nums[MAX_BUFFER_SIZE_ARRAY_SIZE];
+
+        for (unsigned n=0; n<nums_count; n++) {
+            SV** svp = av_fetch(av, n, 0);
+            assert(svp);
+
+            UV value = exs_SvUV(*svp);
+            if (value > UINT32_MAX) {
```

Comment:
So, I looked at the source … these are actually defined as U64, not U32.

I’ll just create a follow-up PR to address this & other stuff.

Reply:
> these are actually defined as U64, not U32.

Yes, but that was my mistake. NFS protocol has only room for U32. See https://github.com/sahlberg/libnfs/pull/408.

Reply:
Ahh. 👍